.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,45 @@
+name: Bug report
+description: Report a bug in Godot
+body:
+
+- type: markdown
+  attributes:
+    value: |
+      - Write a descriptive issue title above.
+      - Search [open](https://github.com/godotengine/godot/issues) and [closed](https://github.com/godotengine/godot/issues?q=is%3Aissue+is%3Aclosed) issues to ensure it has not already been reported.
+      - Verify that you are using a [supported Godot version](https://docs.godotengine.org/en/stable/about/release_policy.html).
+
+- type: input
+  attributes:
+    label: Godot version
+    description: Specify the Git commit hash if using a development or non-official build.
```

Comment:
```suggestion
    description: Specify the Git commit hash if using a development or non-official build. If you use a custom build, please test if your issue is reproducible in official builds too.
```

Reply:
Note for later: Once 3.3.3 is released (with #48450 cherry-picked) and well-established, we could simplify this description by telling users to copy the version string from the project manager or editor.